src/components/ui/QuestionBox/QuestionBox.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
```

Comment:
Our feature was only to create a Question button component.
Our component here is doing too much. It is also handling the logic and also displaying the button. 
We should also decouple our components and it should do only one thing, and do one thing really well. 
When our component does multiple things, thats when the potential bugs can occur. 
We should create a component that accepts via props a color, and also that component should accept a callback function when you click on it. Later on we will do the logic.
This way we have a truly reusable component that we can put anywhere in our application.